staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -161,7 +162,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	StorageVersionHash:      {Default: true, PreRelease: featuregate.Beta},
 	WinOverlay:              {Default: false, PreRelease: featuregate.Alpha},
 	WinDSR:                  {Default: false, PreRelease: featuregate.Alpha},
-	WatchBookmark:           {Default: true, PreRelease: featuregate.Beta},
+	WatchBookmark:           {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.21
```

Comment:
Remove what in 1.21? We shouldn't need to wait that long before removing the feature gate itself?

Reply:
Removed